lerna.json

```diff
@@ -2,5 +2,6 @@
   "packages": [
     "packages/*"
   ],
-  "version": "4.0.0"
+  "version": "4.0.0",
+  "exact": true
```

Comment:
This makes it so that when we run `lerna add`, it defaults to the exact version of the package.